inference-engine/tests/functional/shared_test_classes/src/subgraph/multiple_input.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2021 Intel Corporation
```

Comment:
I'd add fake quantize to the file name since the test is related to the case with fake quantize layers

Reply:
Files were renamed to multiple_input_fq.*.